translations/monero-core_da.ts

```diff
@@ -1,2822 +1,2812 @@
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="267"/>
-        <source>This records rings used by outputs spent on Monero on a key reusing chain, so that the same ring may be reused to avoid privacy issues.</source>
-        <translation>Denne optager ringe der bliver brugt af outputs på Monero på en nøgle-genbrugene blockchain, så den samme ring kan blive genbrugt for at undgå anonymitets problemer.</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="278"/>
-        <source>Key image</source>
-        <translation>Nøgle billede</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="279"/>
-        <source>Paste key image</source>
-        <translation>Indsæt nøgle billede</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="299"/>
-        <source>Get ring</source>
-        <translation>Hent ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="311"/>
-        <source>Get Ring</source>
-        <translation>Hent Ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="317"/>
-        <source>No ring found</source>
-        <translation>Ingen ring fundet</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="334"/>
-        <source>Set ring</source>
-        <translation>Sæt ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="346"/>
-        <source>Set Ring</source>
-        <translation>Sæt Ring</translation>
-    </message>
-    <message>
-        <source>I intend to spend on key-reusing fork</source>
-        <translation type="vanished">Jeg har hensigt at bruge på en nøgle-genbrugene fork(s)</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="378"/>
-        <source>I might want to spend on key-reusing fork(s)</source>
-        <translation>Jeg har nok hensigt at bruge på en nøgle-genbrugene fork(s)</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="392"/>
-        <source>Relative</source>
-        <translation>Relative</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="408"/>
-        <source>Segregation height:</source>
-        <translation>Adskilleses højde:</translation>
-    </message>
-</context>
-<context>
-    <name>Sign</name>
-    <message>
-        <location filename="../pages/Sign.qml" line="60"/>
-        <source>Good signature</source>
-        <translation>God signatur</translation>
-    </message>
-    <me
```

Comment:
forgot `>` after `<translation`. This is breaking the build

Reply:
It's not github. I checked locally.

if you have done it locally, push using `git push -f` to force it to the remote repo

Reply:
Github shows me the lines from 1461-1464 on both your comments. I checked my file and I don't see any missing `>` . Checked my Github repo online and no missing `>` at those lines.

Reply:
Ah ok, sorry i misunderstood what you meant. yes, github is showing the wrong lines. The correct ones are 121 and 921

Reply:
Looked at the file again and I don't see any missing `>` at 121 + 921

Reply:
![121](https://user-images.githubusercontent.com/28106476/45353322-d1035080-b5ba-11e8-8991-8b515e5c8fcf.png)
![921](https://user-images.githubusercontent.com/28106476/45353320-d1035080-b5ba-11e8-8e72-2d5360931585.png)
Also, as you can see, at the bottom of this page, [buildbot is failing](https://build.getmonero.org/builders/monero-gui-ubuntu-amd64/builds/750/steps/compile/logs/stdio) with the reason `lrelease error: Parse error at ../translations/monero-core_da.ts:121:21: Expected '>' or '/', but got '&'.`

Reply:
![capture](https://user-images.githubusercontent.com/33448819/45389070-240cf000-b61b-11e8-9c59-3f7e32f78b84.PNG)

I don't know what is going on. Mine is fine

Reply:
You're right... fixed now. My dark theme was a little too dark and my eyes had mistaken some numbers for 121 + 921... 

Really sorry.

Reply:
I clicked push 5 times in the GUI but oh well will try and do it CLI then

Reply:
MB, fixed.